src/amazonsqs/Module.md

```diff
@@ -147,9 +145,7 @@ public function main(string... args) {
         accessKey: "AKIAIOSFODNN7EXAMPLE",
         secretKey: "wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY",
```

Comment:
All sensitive data should delete from the configuration file.

Reply:
Deleted. Anyway no security threat identified as these are just sample values.